EMRM.py

```diff
@@ -51,65 +67,85 @@ def __init__(self, characters, wlen, input_df):
         self.pos_word_chr = self.get_onehot_chr_rep()
 
     def get_onehot_chr_rep(self):
-        """Get one-hot representation of each character at each position for each word."""
+        """Get one-hot representation of each character at each position
+           for each word.
+        """
         pos_word_chr = np.zeros([self.wlen, self.vocab_size, self.nchar])       
         for w in self.vocab_dict:
             for pos, ch in enumerate(w):
-                pos_word_chr[pos, self.vocab_dict[w], self.characters.index(ch)] = 1
+                pos_word_chr[pos, \
+                             self.vocab_dict[w], \
+                             self.characters.index(ch)] = 1
         return(pos_word_chr)
 
 class OneVirtualReader:
     """ A virtual reader with a vocabulary and some visual properties.
 
     Attributes:
-        vocabulary: A Vocabulary class storing language knowledge.
-        sigma_scale: A float that controls shape of visual acuity function; small number indicates narrow acuity function.
-        Lambda_scale: A float that controls overall visual input quality; small number indicates poor visual input.
-        lpos_range: A list of numbers indicating landing position range.
-        c, B, A: Dictionaries storing pre-computed parameters for computing visual input samples.
+        vocabulary:
+          A Vocabulary class storing language knowledge.
+        sigma_scale:
+          A float that controls shape of visual acuity function;
+          small number indicates narrow acuity function.
+        Lambda_scale:
+          A float that controls overall visual input quality;
+          small number indicates poor visual input.
+        fix_loc_list:
+          A list of numbers indicating landing position.
```

Comment:
This seems unclear to me. Can you try to clarify? It looks from usage like this should be a two-element list where the first element is the minimum possible fixation location and second is the maximum? Can/should these include negative numbers or positions past the word?